### ISSUE TITLE
btcctl tries to make btcd.conf when using wallet

### ISSUE THREAD

**Issue by carryforward** [btcsuite/btcd#875]
ported from https://github.com/btcsuite/btcwallet/issues/460

when I use btcctl to access a wallet, it tries to make a btcd.conf in the .btcd directory. I do not have btcd on the same computer as the wallet, and automatically making a config file is silly when I am not even trying to access the program that uses it.

```
btcctl --wallet --testnet -u username -P password listaccounts
Error creating a default config file: open /home/factom/.btcd/btcd.conf: no such file or directory
```
perhaps if I were trying to control btcd, this would make sense, but not with the wallet.

**Comment by davecgh** [comment]
Nice catch.  It shouldn't try to create `btcctl.conf` when the `--wallet` flag is set

**Comment by davecgh** [comment]
Fixed by #972
